apps/console/src/public/resources/iframe.html

```diff
@@ -0,0 +1,46 @@
+<!--
+~    Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+~
+~    This software is the property of WSO2 LLC. and its suppliers, if any.
+~    Dissemination of any information or reproduction of any material contained
+~    herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+~    You may not alter or remove any copyright or other notice from copies of this content."
+-->
+
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Impersonate Iframe</title>
+    </head>
+    <body onload="authenticate()">
+        <script>
+            var hash = window.location.hash;
+            const params = new URLSearchParams(window.location.search);
+            const userId = params.get("userId");
+            const code_challenge = params.get("codeChallenge");
+            const client_id = "Pfao0gjJ07be0gf29PzEcLFKmUIa";
+            const authorization_endpoint = sessionStorage.getItem("authorization_endpoint");
+
+            function authenticate() {
+
+                const requestUrl = authorization_endpoint
+                    + "?client_id=" + client_id +
+                    "&redirect_uri=" + window.location.origin + window.location.pathname +
+                    "&state=sample_state&scope=internal_user_impersonate&response_type=id_token%20subject_token" +
+                    "&requested_subject=" + userId +
+                    "&nonce=2131236&code_challenge=" + code_challenge + "&code_challenge_method=S256";
+                
+                if (hash === null || hash === "") {
+                    window.location.href = requestUrl;
+                } else {
+                    if (sessionStorage.getItem("impersonation_artifacts") === null) {
+                        sessionStorage.setItem("impersonation_artifacts", hash);
```

Comment:
This is the id token fragment, isn't it? if it is, let's use a better name for the session storage key

Reply:
it contains id_token & subject_token this is why I named it as impersonation_artifacts.